src/EnergyPlus/Data/EnergyPlusData.hh

```diff
@@ -85,6 +85,29 @@ struct DataGlobal : BaseGlobalStruct
 
 struct ExteriorEnergyUseData : BaseGlobalStruct
 {
+    // MODULE PARAMETER DEFINITIONS:
+    int const ElecUse = 1;         // Electricity

```

Comment:
Whenever possible, when we have a group of `int const` in EnergyPlus code, these should be converted over to an `enum` or `enum class`.  They aren't actually 'data', they are basically just another 'type'.  As I mentioned in the previous comment, I am leaning toward leaving structs in their original header locations.  And these converted `int const` into `enums` should be left in there as well.  In fact, in a bunch of cases, you'll probably find they don't need to be in the header at all, and can just be put at the top of the cc file as a nice bonus.

(after further review below, I see the fuel type is used in the unit test, so the enum will need to stay in the header, no problem.)